tools/cli/commands/connect.py

```diff
@@ -52,6 +52,15 @@
     $ {0} {1} example-instance --zone us-central1-a""")
 
 
+# The list of web browsers that we don't want to automatically open.
+#
+# This is a subset of the canonical list of python browser types
+# defined here: https://docs.python.org/2/library/webbrowser.html
+unsupported_browsers = [
```

Comment:
Instead of a blacklist, could we have a list of supported browsers - Chrome, FF, Edge, Safari?